source/Weapon.cpp

```diff
@@ -158,6 +158,8 @@ void Weapon::LoadWeapon(const DataNode &node)
 				inaccuracy = value;
 			else if(key == "turret turn")
 				turretTurn = value;
+			else if(key == "arc")
+				maxAngle = max(0., value);
```

Comment:
what do we need this for?
in the PR description I do not see anything about a new weapon attrib?

Reply:
It's for one on outfit-defined arc. I might've forgot to mention it in the description.

Reply:
Letting outfits also define limits is wholly fabulous :)

Does one override the other, or does it always take the most restrictive of the two?

Reply:
Should be the most restrictive of the two, I would expect.

Reply:
Yeah it should be the most restrictive of the two.

Reply:
Cool, so we agree this is a nice feature. (And the code does indeed seem to check for the most restrictive one.)